src/languages/elixir.md

```diff
@@ -0,0 +1,135 @@
+# Elixir
+
+Platform.sh supports building and deploying applications written in Elixir.  They are automatically compiled during the Build phase, and support both committed dependencies and download-on-demand. The underlying Erlang version is 
+22.0.7.
+
+## Supported versions
+
+* 1.9
+
+To specify an Elixir container, use the `type` property in your `.platform.app.yaml`.
+
+```yaml
+type: 'elixir:1.9'
+```
+
+## Assumptions
+
+Platform.sh is making assumptions about your application to provide a more streamlined experience. These assumptions are the following:
+
+- Your `mix.exs` file is at the root of your application.
+
+Platform.sh will then run `mix do deps.get, deps.compile, compile` on your application to build a binary.
+
+If you don't want these assumptions, you can disable this behavior by specifying in your `.platform.app.yaml`:
+
+```yaml
+build:
+    flavor: none
+```
```

Comment:
Wait, why are we using a flavor here?  We have moved away from it for everything since Node, because it's too much magic.  Why is this back?  That's very inconsistent.

Reply:
Florian wanted a flavour; I was not in favor .. @ralt ... even more so that I need to make it quite magical (ie either two flavours or I need   a big if/else block there).
But I am good either way.